db.go

```diff
@@ -157,6 +158,17 @@ func newDBMetrics(db *DB, r prometheus.Registerer) *dbMetrics {
 		Name: "prometheus_tsdb_retention_cutoffs_failures_total",
 		Help: "Number of times the database failed to cut off block data from disk.",
 	})
+	m.startTime = prometheus.NewGaugeFunc(prometheus.GaugeOpts{
+		Name: "prometheus_tsdb_start_time_seconds",
```

Comment:
`prometheus_tsdb_lowest_timestamp` makes more sense here, as start_time could be confused with the process start time and it may or may not be seconds :)

Reply:
I think we should keep the unit suffix and enforce the metric's value to be in seconds though.

Reply:
No. From a general TSDB perspective, there is no reason that the time passed into `.Append(t int64, v float64)`, is a wall time. It could be anything, and people may even pass nano-seconds.

Reply:
Hmm. Does it make sense to have an "opaque" metric then? Wouldn't it be better to expose the raw value to the caller (eg Prometheus) and let it compute the metric?

Reply:
If Prometheus appends millisecond timestamps to the TSDB, then this metric as collected might be slightly more difficult to work with using something like the built-in `time()` function for example. For that reason I agree that it would be nice to adhere to metric naming best practices and include the base unit as a suffix in the metric name, converting the millisecond timestamp to a unix timestamp if necessary. As @simonpasquier mentioned I would think that instrumenting this metric directly in Prometheus instead would allow for this.

With that being said perhaps there are other consumers of the TSDB where exposing this lowest timestamp metric directly would still be useful.